lib/src/value/calculation.dart

```diff
@@ -120,6 +121,25 @@ class SassCalculation extends Value {
     return SassCalculation._("max", args);
   }
 
+  /// Creates a `sqrt()` calculation with the given [argument].
+  ///
+  /// Each argument must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation]. It must be passed at least one argument.
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value sqrt(Object argument) {
+    argument = _simplify(argument);
+    if (argument is! SassNumber) {
+      return SassCalculation.sqrt(argument);
```

Comment:
`SassCalculation.sqrt` is the current function this is called from, so this infinitely recurses (it's not currently caught by tests because you don't have any where the argument to `sqrt` doesn't simplify to a number).

You should call `SassCalculation._("sqrt", args)` here.